README.md

```diff
@@ -19,36 +19,30 @@ Participating in the Gaffer community requires abiding by the project's [Code of
 
 Compiled binary releases are available for download from the [releases page](https://github.com/GafferHQ/gaffer/releases).
 
+Gaffer currently supports **Linux** (Centos 7) and **macOS** (10.14).
+
 
 ## Building ##
 
 [![Build Status](https://dev.azure.com/GafferHQ/Gaffer/_apis/build/status/GafferHQ.gaffer?branchName=master)](https://dev.azure.com/GafferHQ/Gaffer/_build/latest?definitionId=1&branchName=master)
 
-Gaffer is a fairly large project, and as such has a fairly complex build process. Before you start, make sure you have the following prerequisites installed on your system, which will be used to perform the build itself.
+Gaffer targets the [VFX Reference Platform](https://vfxplatform.com). We are currently on **CY2018**. Aside from general platform development packages, we specifically require the following tools that may not be installed by default on your system. Without these, you will not be able to build Gaffer.
 
+> NOTE: From time to time, this list may change. For a complete, accurate, and up-to-date method of installing the prerequisites on CentOS, refer to the [Docker setup](https://github.com/GafferHQ/build/blob/master/Dockerfile) we use for building automatic releases.
```

Comment:
\> NOTE:
↓
\> \*\*Note:\*\*

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3604/commits/8f41455423ba0863f98f69e61e5eb4a23bd03841